src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/perform_interaction_request.h

```diff
@@ -53,6 +53,10 @@ namespace commands {
 class PerformInteractionRequest
     : public app_mngr::commands::CommandRequestImpl {
  public:
+  /**
+   * @brief Enum, defines which PerformInteraction response came first
+   */
+  enum class FirstPerformInteractionResponser { NONE, UI, VR };
```

Comment:
@YarikMamykin `FirstAnsweredInterface`? May be a bit more clear? What do you think?

Reply:
@AByzhynar , I think it is better to keep `FirstPerformInteractionResponser`, because this enum is related only to VR/UI.PerformInteraction responses. In my opinion, `FirstAnsweredInterface` has more general meaning for this case.

Reply:
@YarikMamykin Name is more clear and it is more general but as you have defined it in PI only - it is still related to PT only. So I still suggest to make name more clear

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/pull/3122/commits/0190836450ab1d7e130bbb8e32315eb3325a9a68